### DISCUSSION TITLE
API versioning in tractusx

### DISCUSSION THREAD

**Discussion by lholthof** [eclipse-tractusx/tractusx-edc#952] [Q&A]
APIs in tractusx (e.g. /erds, /business-partner-groups) are not indicating any version unlike most APIs in upstream. 

Would it make sense to add a version in alignment with upstream's decision record [Consistent Versioning of external APIs](https://github.com/eclipse-edc/Connector/blob/a10ed756e51312000c040a58117e7d08760abe72/docs/developer/decision-records/2023-11-09-api-versioning/README.md)?

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
